en/components/themes/themes-list/bootstrap.md

```diff
@@ -0,0 +1,16 @@
+### Switching from Material to Bootstrap theme
+Since version `9.0` our components support a new theme, built based on the [`Bootstrap 4`](https://getbootstrap.com/) and [`and angular bootstrap`](https://ng-bootstrap.github.io/#/home).  
```

Comment:
```suggestion
Since version `9.0` our components include a new theme inspired by [Bootstrap 4](https://getbootstrap.com/) and [ng-bootstrap](https://ng-bootstrap.github.io/#/home).
```